integration/admin-client/src/main/java/org/keycloak/admin/client/resource/UserResource.java

```diff
@@ -67,6 +68,19 @@
     List<GroupRepresentation> groups(@QueryParam("search") String search,
                                      @QueryParam("first") Integer firstResult,
                                      @QueryParam("max") Integer maxResults);
+    
+    @Path("groups")
+    @GET
+    List<GroupRepresentation> groups(@QueryParam("first") Integer firstResult,
+                                     @QueryParam("max") Integer maxResults,
+                                     @QueryParam("full") @DefaultValue("false") boolean fullRepresentation);
```

Comment:
Please use parameter name `briefRepresentation` to keep consistent with the rest of API

Reply:
Ha yes it's true, I think the full representation is now present already in 2 API's I will fix this one and the other ones.

Reply:
Fixed. Should we fixe all the full parameters in this issue ? because it had common change with other endpoint for the method toRepresentation

Reply:
Thank you! That would be fantastic.

Reply:
I created an other PR, but maybe I should cherry pick the commit to this one and also change the `toRepresentation` method here ? https://github.com/keycloak/keycloak/pull/6655

Reply:
Just to be clear, the proper name is `briefRepresentation`, not `full`

Reply:
Yes I changed it to `briefRepresentation` but some methods inside `ModelToRepresentation.java` was already using a `full` boolean that's why I used `full` in the API at the begining. So for theses PR I just changed the API definition ( that mean that it won t work anymore if someone use `full` already ... but I think there is no documentation about `full` so it should be ok ). So I changed the API definition, not the `ModelToRepresentation.java` method, I can't change both in separate pull request because if one is validate before the other it will broke the other. It's not really clear like that but have a look on the code you will understand.